superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -58,6 +58,7 @@ import {
 import { InfoTooltipWithTrigger } from '@superset-ui/chart-controls';
 import { AlertReportCronScheduler } from './components/AlertReportCronScheduler';
 import { NotificationMethod } from './components/NotificationMethod';
+import { AsyncSelect } from 'src/components';
```

Comment:
There's another import above from `src/components`. To fix the lint error:

`import { AntdCheckbox,  AsyncSelect } from 'src/components';`